bundles/org.openhab.transform.basicprofiles/src/test/java/org/openhab/transform/basicprofiles/internal/profiles/StateFilterProfileTest.java

```diff
@@ -793,6 +793,68 @@ public static Stream<Arguments> testFunctions() {
     @MethodSource
     public void testFunctions(Item item, String condition, List<State> states, State input, boolean expected)
             throws ItemNotFoundException {
+        internalTestFunctions(item, condition, states, input, expected);
```

Comment:
Why did you add `internalTestFunctions` here?? Surely the two test can simply be one after the other?